tests/e2e/framework/framework.go

```diff
@@ -25,6 +25,8 @@ import (
 
 var (
 	skipCleanup = flag.Bool("skip_cleanup", false, "Debug, skip clean up")
+	// TestVM is true if in this test run user wants to test VM on istio
+	TestVM = flag.Bool("test_vm", false, "Wether to test VM on istio")
```

Comment:
try running the linters, they should find the english typo here whether not wether